_pages/calls/papers.md

```diff
@@ -130,6 +130,14 @@ Short paper submissions must describe original and unpublished work. Please note
 
 Short papers may consist of up to 4 pages of content, plus unlimited references and appendix. Upon acceptance, short papers will be given 5 content pages in the proceedings. Authors are encouraged to use this additional page to address reviewers’ comments in their final versions.
 
+### Style Files and Formatting
+
+The templates can be downloaded directly. ([LaTeX]({{ "/downloads/eacl2023_latex.zip" | relative_url }}), [Word]({{ "/downloads/eacl2023.docx" | relative_url }})).
+
```

Comment:
Could you add back the text `Both long and short papers must follow the EACL 2023 two-column format, using the supplied official style files. <b>Please do not modify these style files, nor should you use templates designed for other conferences. Submissions that do not conform to the required styles, including paper size, margin width, and font size restrictions, will be rejected without review.</b>` ? It seems critical in case we reject people because of this and we don't state it explicitly.

Reply:
I've added it back. Can you check if that paragraph makes sense all appended together?

Reply:
Looks good 👍